tests/layer_tests/onnx_tests/test_lstm.py

```diff
@@ -143,6 +143,9 @@ def create_lstm(self, direction: str, cell_type: str, hidden_size=128):
     @pytest.mark.parametrize('direction', ["forward", "bidirectional", "reverse"])
     @pytest.mark.parametrize('cell_type', ["LSTM", "GRU", "RNN"])
     def test_lstm_simple_precommit(self, direction, cell_type, ie_device, precision, ir_version, temp_dir):
+        if ie_device == 'GPU':
+            # TODO: add test to precommit after heavy/light optimization
+            pytest.skip("Sporadic TimeoutError (Inference Engine running timed out) on GPU")
```

Comment:
We usually add a ticket number in the code so we know how to track the progress of the activity. Please, create the ticket to GPU plugin and leave a reference here.

Reply:
I think that this failed test is a consequence of the use of parallelization. In the near future, work will be carried out to divide the scope of tests into heavy and light models (CVS-55862), with different numbers of workers for parallel execution.
After this work, the test will be returned to the scope.